lib/noticed/delivery_methods/fcm.rb

```diff
@@ -23,6 +23,16 @@ def send_notification(device_token)
         end
       end
 
+      def format_notification(device_token)
+        raise ArgumentError, "No json for fcm delivery. Add the 'json' option in 'deliver_by :fcm'." unless (method = config[:json])
```

Comment:
Is this redundant since we already have `required_option :json`?

Reply:
Yep. I missed that. Thanks.

Removing it tidies up that method nicely too